tests/DataFactories/Settings.php

```diff
@@ -41,4 +41,9 @@ function withTrackingDisabled() {
     Setting::setValue('tracking.enabled', false);
     return $this;
   }
+
+  function withNPSPollDisabled() {
```

Comment:
I think that this will be needed in many tests. What about adding this settings into the database dump?

Reply:
It is in the free plugin DB dump. I think any option we choose we should use it in both plugins.

Reply:
Let's copy it to the Premium DB dump then.